packages/mattermost-redux/src/selectors/entities/timezone.ts

```diff
@@ -1,10 +1,16 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {createSelector} from 'reselect';
+
+import {getUsers} from 'mattermost-redux/selectors/entities/users';
+
 import {GlobalState} from 'mattermost-redux/types/store';
+import {IDMappedObjects} from 'mattermost-redux/types/utilities';
+import {UserProfile, UserTimezone} from 'mattermost-redux/types/users';
 
-export function getUserTimezone(state: GlobalState, id: string) {
-    const profile = state.entities.users.profiles[id];
+export function getUserTimezone(users: IDMappedObjects<UserProfile>, id: string): UserTimezone {
```

Comment:
Why not make this just take the singular user we need? That seems like it would make `makeGetUserTimezone` recalculate less often, and we'd probably have the user we want more often anyway.

Reply:
Yeah, that's probably better. I was just looking to change the signature as little as possible